### PR TITLE
Minor clarifications to shell-novice

### PR DESCRIPTION
**Pull request by dme26**
Syntax of "plus 300x299/2 runs of goodiff" might not be understood by all. I'd suggest use of UTF-8 or HTML entity forms of multiplication and division symbols, but for ASCII safety have rewritten the expression in words for the moment (I'm happy to update pull request for UTF-8 or HTML entities, of course).

Also clarified that the pipe construction exercise references "animals.txt" content from the previous exercise (other exercises on that page are self-contained).

[PS in future I'd use two separate pull requests, but I'd already done the second commit by the time I realised I'd forgotten to use two branches. Easier to continue than to rebase. Apologies!]

### CODE REVIEW

File: 00-intro.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ this step will "only" take about two weeks.
 
 The bad news is that if she has to run `goostat` and `goodiff` by hand,
 she'll have to enter filenames and click "OK" 45,150 times
-(300 runs of `goostat`, plus 300x299/2 runs of `goodiff`).
+(300 runs of `goostat`, plus half 300 times 299 runs of `goodiff`).
```
Comment (shwina) [inline 00-intro.md]: How about "300*299/2 (half of 300 times 299)"

File: 03-pipefilter.md [hunk 2]
```diff
@@ -508,7 +508,7 @@ so this matches all the valid data files she has.
 
 > ## Pipe construction {.challenge}
 >
-> The command:
+> If the file called `animals.txt` contains the data described in the previous exercise, the command:
```
Comment (shwina) [inline 03-pipefilter.md]: I'd propose:

> For the file `animals.txt` from the previous exercise, ...

**Comment by shwina** [comment]
Thank @dme26 - good catches. Just a little pedantry from me - please let me know what you think.

File: 00-intro.md (hunk 1 above)
Comment (dme26) [inline 00-intro.md]: @shwina yes, your suggestion is ideal—I'll update the pull request.

File: 03-pipefilter.md (hunk 2 above)
Comment (dme26) [inline 03-pipefilter.md]: Sounds good to me.

**Comment by dme26** [comment]
Your suggestions are ideal—I’ve updated the pull request now. Thanks!

**Comment by shwina** [comment]
Thanks!

**Pull request merged**